code/datums/components/parry.dm

```diff
@@ -42,6 +46,20 @@
 	else
 		parryable_attack_types = list(_parryable_attack_types)
 
+	// From the attack type defines in `code\__DEFINES\combat.dm`.
+	var/static/list/attack_types_english = list(
+		1 = "melee attacks",
+		2 = "unarmed attacks",
+		3 = "projectiles",
+		4 = "thrown projectiles",
+		5 = "leap attacks"
+	)
```

Comment:
couldnt you make this use the defines instead the of numbers?

Reply:
Yeah, I don't know why I used numbers.